packages/calendar/test/todolist-1.spec.js

```diff
@@ -0,0 +1,221 @@
+function getPossibleExpressions(expressionType, count) {
+    const data = require('../data/expressions/en.json')
+    switch (expressionType) 
+    { 
+        case "create list":
+            return data.todolist.create_list.expressions
+        case "view lists":
+            return data.todolist.view_lists.expressions
+        case "view list":
+            return data.todolist.view_list.expressions
+        case "rename list":
```

Comment:
Instead to return the same things for multiple values of `expressionType`, you can do something like that :
```js
case "view lists":
case "view list":
  return data.todolist.view_list.expressions
```